ember-cli-build.js

```diff
@@ -0,0 +1,25 @@
+var EmberApp = require('ember-cli/lib/broccoli/ember-addon')
+
+module.exports = function (defaults) {
+  var app = new EmberApp(defaults, {
+    'ember-cli-mocha': {
+      useLintTree: false
+    },
+    sassOptions: {
+      includePaths: [
+        'bower_components/bourbon/app/assets/stylesheets'
```

Comment:
So, we don't have any mixins we're using from boubon that aren't related to prefixing?